### PR TITLE
Give users more details to make an educated decision.

### PR DESCRIPTION
**Pull request by davidbuzz**

### CODE REVIEW

File: common/source/docs/common-install-gcs.rst [hunk 1]
```diff
@@ -11,11 +11,11 @@ If you just want to fly you may instead choose to use a GCS running on a mobile
 
 Installation links for some of the more popular GCS systems are provided below:
 
-- :ref:`Mission Planner <planner:home>` (Windows): :ref:`Install Mission Planner <planner:install-mission-planner>`
+- :ref:`Mission Planner <planner:home>` (Windows, Linux, Android): :ref:`Install Mission Planner <planner:install-mission-planner>` This is the most fully-featured and constantly updated GCS that we have , and is recommended for first-time users and power users alike.    It does all the things, best on Windows.
```
Comment (Hwurzburg) [inline common/source/docs/common-install-gcs.rst:14]:
```suggestion
- :ref:`Mission Planner <planner:home>` (Windows, Linux, Android): :ref:`Install Mission Planner <planner:install-mission-planner>` This GCS is the most compatible and closely tracks new features and updates in ArduPilot. It is recommended for first-time users and power users alike.  It functions best in the Windows environment.
```

File: common/source/docs/common-install-gcs.rst [hunk 2]
```diff
@@ -11,11 +11,11 @@ If you just want to fly you may instead choose to use a GCS running on a mobile
...
 
-- `QGroundControl <http://qgroundcontrol.com/>`__ (Windows, Mac OS X, Linux, Android and iOS):
+- `QGroundControl <http://qgroundcontrol.com/>`__ (Windows, Mac OS X, Linux, Android and iOS). Not bad, many people like it, seems to have less complete support for ArduPilot drones than MP (above), but does the basics well on all platforms.
```
Comment (Hwurzburg) [inline common/source/docs/common-install-gcs.rst:16]:
```suggestion
- `QGroundControl <http://qgroundcontrol.com/>`__ (Windows, Mac OS X, Linux, Android and iOS).This is the most popular GCS for Android based systems and provides the best presentation on small format devices like cellphones. Does not have all the capabilities of Mission Planner, but has a more user-friendly interface for some functions. Operates well on all platforms
```

File: common/source/docs/common-install-gcs.rst [hunk 3]
```diff
@@ -11,11 +11,11 @@ If you just want to fly you may instead choose to use a GCS running on a mobile
...
 
    - `Download and Install <https://docs.qgroundcontrol.com/en/getting_started/download_and_install.html>`__ (stable)
    - `Daily builds <https://docs.qgroundcontrol.com/en/releases/daily_builds.html>`__
    
-- :ref:`APM Planner 2 <planner2:home>` (Windows, Mac OS X, Linux): :ref:`Install APM Planner 2 <planner2:installing-apm-planner-2>`
+- :ref:`APM Planner 2 <planner2:home>` (Windows, Mac OS X, Linux): :ref:`Install APM Planner 2 <planner2:installing-apm-planner-2>` - Originally a fork of QGC for a cross-platform GCS when MP wasn't.  Software not updated much these days, not as big of a following, seems to do 'the basics' ok for some people.
```
Comment (Hwurzburg) [inline common/source/docs/common-install-gcs.rst:21]:
```suggestion
- :ref:`APM Planner 2 <planner2:home>` (Windows, Mac OS X, Linux): :ref:`Install APM Planner 2 <planner2:installing-apm-planner-2>` -Provides basic functionality, but is not currently well maintained and updated.
```

**Review by Hwurzburg — CHANGES_REQUESTED** [review]

**Comment by khancyr** [comment]
A little too much engaged ! Henry suggestions are welcome !

**Comment by davidbuzz** [comment]
Henry, i don't have a strong opinion about any of the wording.   I've accepted all your suggestions.

**Review by Hwurzburg — APPROVED** [review]
thanks...merging

**Pull request merged**